xonsh/xonfig.py

```diff
@@ -47,7 +47,8 @@
     color_style,
 )
 from xonsh.foreign_shells import CANON_SHELL_NAMES
-from xonsh.xontribs import xontrib_metadata, find_xontrib, xontribs_loaded
+from xonsh.xontribs import find_xontrib, xontribs_loaded
+import xonsh.xontribs_meta
```

Comment:
Amalgamate is failing because this line needs to be the following:

```
from xonsh.xontribs_meta import Xontrib, get_xontribs
```

Reply:
Thanks updated.